store/index.js

```diff
@@ -275,12 +305,29 @@ export const getters = {
             }
         };
 
+        // 1. Fill out Annotations 'IsAbout' section
+
+        // A. Label matches the assigned category
         formattedOutput.Annotations.IsAbout.Label = category;
 
+        // B. Term matches the category identifier
         formattedOutput.Annotations.IsAbout.TermURL = p_state.categories[category].identifier;
 
+        // 2. Save the transformation heuristic from the list of possible heuristics
         formattedOutput.Annotations.Transformation = p_state.transformationHeuristics[annotatedDictColumn.transformationHeuristic];
 
+        // 3. Save the missing values
+        formattedOutput.Annotations.MissingValues = annotatedDictColumn.missingValues;
```

Comment:
To adhere to the [exisitng documentation](https://www.neurobagel.org/documentation/dictionaries/#age), I think it's better to remove the `MissingVlaues`.

Reply:
Not quite, let's ask @surchs.

Reply:
What we discussed:
1) MissingValues is actually implemented for continuous values, it is just missing a test to show that feature -> issue #
2) Since MissingValues for continuous values is OK with the spec (and the CLI should know what to do with it), we will include MissingValues in the output of the annotator (and also test that case)
3) To prevent future issues around ambiguity of "what's in the spec" and so on, we opened issue https://github.com/neurobagel/bagel-cli/issues/144

Reply:
MissingValues will remain unless schema documentation changes to state otherwise.